pkg/restapi/common/store/tokens/store.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package tokens
+
+import (
+	"fmt"
+	"github.com/trustbloc/edge-agent/pkg/restapi/common/store"
```

Comment:
same here, separate core from third party libraries with a blank line.. the linter should catch this formatting error.

also edge-agent should be the last import because it's a local import, edge-core is third party and should be in the middle, separate with a blank line as well.